docs-new/docs/graphql/core/databases/citus-hyperscale-postgres/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+description: Hasura Citus - Hyperscale database support
+keywords:
+  - hasura
+  - docs
+  - databases
+  - citus
+  - hyperscale
+---
+# Citus / Hyperscale
```

Comment:
Please leave an empty space before new section

Not only for sections, basically for anything markdown thats in new line.